src/client/content_cao.cpp

```diff
@@ -1432,6 +1449,17 @@ void GenericCAO::updateBonePosition()
 			bone->updateAbsolutePosition();
 		}
 	}
```

Comment:
This is almost line for line what irrlicht does in CAnimatedMeshSceneNode, in its EJUOR_READ path. Due to an oversight, the other code paths do not do this, and some lines above we've just set EJUOR_CONTROL so we're sure to miss it.